smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/AmqpConnector.java

```diff
@@ -247,6 +220,10 @@ public Flow.Publisher<? extends Message<?>> getPublisher(Config config) {
 
         AmqpFailureHandler onNack = createFailureHandler(ic);
 
+        if (tracing && amqpInstrumenter == null) {
+            amqpInstrumenter = AmqpOpenTelemetryInstrumenter.createForConnector();
```

Comment:
Moving this here is my biggest doubt. How would I go about getting the value of `tracing` from the @PostConstruct method?